src/Uno.UWP/Devices/Bluetooth/BluetoothLEDevice.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Windows.Devices.Bluetooth
+{
+	public partial class BluetoothLEDevice : global::System.IDisposable
+	{
+		private static string _deviceSelectorPrefix = "System.Devices.DevObjectType:=5 AND System.Devices.Aep.ProtocolId:=\"{BB7BB05E-5972-42B5-94FC-76EAA7084D49}\" AND ";
+		private static string _deviceSelectorIssueInquiry = "System.Devices.Aep.Bluetooth.IssueInquiry:=System.StructuredQueryType.Boolean";
+
+		public static string GetDeviceSelector()
+		{
+			return _deviceSelectorPrefix + "(System.Devices.Aep.IsPaired:=System.StructuredQueryType.Boolean#True OR " + _deviceSelectorIssueInquiry + "#False)";
+		}
+
+		public static string GetDeviceSelectorFromPairingState( bool pairingState)
+		{ 
+			if (pairingState)
+			{
+				return _deviceSelectorPrefix + "(System.Devices.Aep.IsPaired:=System.StructuredQueryType.Boolean#True OR " + _deviceSelectorIssueInquiry + "#False)";
+			}
+			else
+			{
+				return _deviceSelectorPrefix + "(System.Devices.Aep.IsPaired:=System.StructuredQueryType.Boolean#False OR " + _deviceSelectorIssueInquiry + "#True)";
+			}
+		}
+
+		public static string GetDeviceSelectorFromConnectionStatus( BluetoothConnectionStatus connectionStatus)
+		{ 
+			if (connectionStatus == BluetoothConnectionStatus.Connected)
+			{
+				return _deviceSelectorPrefix + "(System.Devices.Aep.IsConnected:=System.StructuredQueryType.Boolean#True OR " + _deviceSelectorIssueInquiry + "#False)";
+			}
+			else
+			{
+				return _deviceSelectorPrefix + "(System.Devices.Aep.IsConnected:=System.StructuredQueryType.Boolean#False OR " + _deviceSelectorIssueInquiry + "#True)";
+			}
+		}
+
+		public static string GetDeviceSelectorFromDeviceName( string deviceName)
+		{
+			return _deviceSelectorPrefix + "(System.ItemNameDisplay:=\"" + deviceName + "\" OR " + _deviceSelectorIssueInquiry + "#True)";
+		}
+
+		public static string GetDeviceSelectorFromBluetoothAddress( ulong bluetoothAddress)
+		{
+			string macAddr = string.Format("{0:x12}", bluetoothAddress);

```

Comment:
Please do that the check notice mentioned.

Reply:
Sorry, I don't understand. Can you rephrase this?

Reply:
I updated my comment.